.ci/README.md

```diff
@@ -26,7 +26,7 @@ export VAULT_ADDR=YOUR_VAULT_INSTANCE_ADDRESS
 export GITHUB_TOKEN=YOUR_PERSONAL_ACCESS_TOKEN
 ``` 
 
-Per repro, depending on the job, set up .env and deployer-config.yml files. E.g.: to repro e2e tests run, look at its [Jenkinsfile](e2e/Jenkinsfile) and rerun the script locally in repo root: 
+Per repro, depending on the job, set up .env and deployer-config.yml files by looking at [setenvconfig](setenvconfig):
 ```
 cat >.env <<EOF
```

Comment:
Not really, because the script also uses some variables set by Jenkins  like `BUILD_NUMBER`:
```sh
> .ci/setenvconfig e2e/master
.ci/setenvconfig: ligne 150: BUILD_NUMBER : variable sans liaison
                                                                                                                                         
> BUILD_NUMBER=1 .ci/setenvconfig e2e/master
.ci/setenvconfig: ligne 150: VAULT_ROLE_ID : variable sans liaison
                                                                                                                                         
> VAULT_ROLE_ID=x BUILD_NUMBER=1 .ci/setenvconfig e2e/master
.ci/setenvconfig: ligne 150: VAULT_SECRET_ID : variable sans liaison
                                                                                                                                         
> VAULT_SECRET_ID=x VAULT_ROLE_ID=x BUILD_NUMBER=1 .ci/setenvconfig e2e/master
```

Reply:
Ah, yes, you are right. Originally I thought to move the common ones to the paragraph above, but now I have another idea.

What do think about moving some "detect if it's local run" logic from `.ci/Makefile` to `setenvconfig`? `$VAULT_ROLE_ID` and `$VAULT_SECRET_ID` won't be there if that's the case and  `$BUILD_NUMBER` will need to be faked anyway. I think I'd propose to either:
1. Detect if it's a local run using the same env var as we do in the `.ci/Makefile` (`ifdef BUILD_ID`) and set vault stuff to empty and `$BUILD_NUMBER` to `local` or `local-{date}`, or
2. Go a step further and move the code from `.ci/Makefile` that invokes vault and "fixes" `deployer-config.yaml` with right `roleId`.

I think I'd lean towards 1. because it doesn't make `setenvconfig` have any dependencies.

If we do that then we can do the below for every job we have (+ set the additional vars if there is any input):
```
.ci/setenvconfig {whatever job we want}
make -C .ci {whatever respective Jenkinsfile contains here}
```

WDYT? We can chat offline too.

Reply:
Good idea. I also prefer 1. to keep `setenvconfig` simpler, without the `vault` dependency.